fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -497,6 +529,7 @@ void FixedLagSmoother::transactionCallback(
       // ...check if we should
       if (sensor_models_.at(sensor_name).ignition)
       {
+        ROS_INFO_STREAM("Ignition occured");
```

Comment:
I have no opinions here whatsoever. Just asking questions:
Do you like this new log line printed all the time? Or would DEBUG be a better fit? Should the same message be implemented in the batch optimizer?

Reply:
Whoops, left this in by accident. I was testing... something, I don't remember what, and forgot to remove this. Doing that now